### ISSUE TITLE
"Last refresh:" activity status message is not treated as default status

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-client#414]
## Description

When downloading a file, you'll see the "Downloading file..." message in the activity status bar. After it finishes, the "Last refresh" message does not reappear. It's blank until the next 5-minute refresh happens.

## STR

1. Download a file and see message in status bar
2. Wait until message goes away (~10 seconds)
3. See no "Last refresh" until the 5-minute sync occurs or until you just manually click the refresh button

## Expected behavior

By default, we should always show the last time a refresh occurred.

## Idea

* The `ActivityStatusBar` could keep a timer for each message and emit a signal when the message times out to the controller. Then the controller could grab the last time since a refresh and set the status of the activity bar.

**Comment by eloquence** [comment]
Confirmed in local dev env.

**Comment by eloquence** [comment]
In the UX meeting today, we discussed significant potential revisions to the refresh widget, largely to ensure the UI remains consistent with the changes described in #652. In addition to changing the refresh widget to an activity indicator, the specific proposal we discussed was to show a timestamp only in case the client is offline or experiencing network issues, since the client can otherwise be expected to constantly poll the server for new messages/replies. 

This issue is not currently actionable, so removing form the backlog for now (but kept open because it's still a bug); added a card to create a spec for the intended behavior.
